include/DecoObjects/DecoObject.hpp

```diff
@@ -24,6 +24,8 @@ this program.  If not, see <http://www.gnu.org/licenses/>. */
 
 class DecoObject {
     public:
+        virtual ~DecoObject() = default;
```

Comment:
Hmmm... I think this code base isn't using C++11 yet. Since these fixes are to align with SFML 2, it might make sense to just stick to `{}` for now. The SFML 3 update will need a forced C++17 update anyways.